### DISCUSSION TITLE
Trend Micro Security blocks RustDesk 1.2.3 client

### DISCUSSION THREAD

**Discussion by jrmain** [rustdesk/rustdesk#6740] [General]
### Bug Description

Tried to download the RustDesk client on a Windows 10 PC running Trend Micro Security, and the downloaded file (rustdesk-1.2.3-x86_64.exe) was deleted immediately. Unable to allow or whitelist the file, I had to disable Trend completely to get RustDesk to download and run. I'm not a fan of Trend but it's a work computer so it's not my call.

### How to Reproduce

Try to download the RustDesk client on a Windows 10 PC running Trend Micro Security (eg. rustdesk-1.2.3-x86_64.exe).

### Expected Behavior

The downloaded file should be allowed to exist.

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.2.3 -> n/a

### Screenshots

No relevant screenshot.

**Comment by Syntaxrabbit** [comment]
Same problem on Windows11, same crappy snake oil anti virus solution from TrendMicro. Rustdesk is a daily driver for me.
Maybe something not OK with the digital signature?

![grafik](https://github.com/rustdesk/rustdesk/assets/63979422/51d63cb3-d909-43e3-9df4-f6a43cf57200)
